### PR TITLE
Alarm statuss

### PR DESCRIPTION
**Pull request by brian-smith723**
@Dan I have no idea why there are so many files. Let me know if you need to delete something. Might want to check with @dlangston , too. The filtering works.

### CODE REVIEW

**Comment by oceanzus** [comment]
It looks like .gitignore needs to be update to ignore your .metadata folder.

File: ooiui/core/routes/common.py [hunk 1]
```diff
@@ -252,9 +256,8 @@ def get_operator_event_type():
     response = requests.get(app.config['SERVICES_URL'] + '/operator_event_type', params=request.args)
     return response.text, response.status_code
 
-
 @app.route('/api/uframe/glider_tracks', methods=['GET'])
 def get_glider_track():  
     token = get_login()
     response = requests.get(app.config['SERVICES_URL'] + '/uframe/get_glider_track/'+request.args['id'], auth=(token, ''), data=request.args)
-    return response.text, response.status_code
```
Comment (oceanzus) [inline ooiui/core/routes/common.py]: Why was this line removed?

File: Gruntfile.js [hunk 2]
```diff
@@ -203,7 +203,29 @@ module.exports = function(grunt) {
             "ooiui/static/js/partials/Event.html",
             "ooiui/static/js/partials/LoggedInNavItems.html"
           ],
-          "ooiui/static/js/partials/compiled/landingPages.js": [
```
Comment (oceanzus) [inline Gruntfile.js]: Was landingpages.js not needed anymore?

File: ooiui/static/js/partials/StatusUIAccordion.html [hunk 3]
```diff
@@ -0,0 +1,196 @@
...
+<% var yellowBG = 'dogTagBtn well yellow-backgroud-gradiant ' %>
+<% var redBG = 'dogTagBtn well red-backgroud-gradiant' %>
+<% var unknownBG = 'dogTagBtn well unknown-backgroud-gradiant' %>
+
+<% var blueBadge = 'small-status-icon-ok' %>
+<% var yellowBadge = 'small-status-icon-issue' %>
+<% var redBadge =  'small-status-icon-error' %>
+<% var unknownBadge = 'small-status-icon-unknown' %>
+
+<!-- ==========================================((((((((((())))))))))))==================================================== -->
```
Comment (oceanzus) [inline ooiui/static/js/partials/StatusUIAccordion.html]: This code makes me think it was "borrowed" from another project.  If so, what was the source project?

Comment (brian-smith723) [inline ooiui/static/js/partials/StatusUIAccordion.html]: I pulled down danL 's branch and started working on the filtering for the alarms page. I guess DanL would be the source.

File: ooiui/core/routes/common.py (hunk 1 above)
Comment (DanielJMaher) [inline ooiui/core/routes/common.py]: That was only removed because he hadnt updated to your most recent changes... he needs to update that (rebase?)

@brian-smith723 You need to include only the files and lines of code you want to commit.. you've included a file/line that you dont want in here.

File: Gruntfile.js (hunk 2 above)
Comment (DanielJMaher) [inline Gruntfile.js]: I think this was accidental

File: ooiui/static/js/partials/StatusUIAccordion.html (hunk 3 above)
Comment (DanielJMaher) [inline ooiui/static/js/partials/StatusUIAccordion.html]: @dlangston Need to get some input here.  Where was this from?  @birdage code?

Comment (brian-smith723) [inline ooiui/static/js/partials/StatusUIAccordion.html]: I can  take the changes I've made and put them on a different branch with the most recent code.

**Comment by oceanzus** [comment]
I concur with that approach. This PR is just too messy.  I believe Dan will concur, close this PR and we will deal with it on Monday with a new branch and just your changes.

**Comment by DanielJMaher** [comment]
@brian-smith723 I have a phone call at 9 AM, but I will be around shortly after to chat about making a new PR
